### ISSUE TITLE
How to copy the text?

### ISSUE THREAD

**Issue by goldwind-ting** [emilk/egui#2033]
```rust
ScrollArea::vertical().auto_shrink([false; 2]).show_rows(
                ui,
                row_height,
                num_rows,
                |ui, row_range| {
                    for row in row_range {
                        if row == 0 {
                            ui.columns(3, |columns| {
                                columns[0].label("Desc");
                                columns[1].label("Account");
                                columns[2].label("Password");
                            });
                        } else {
                            if row - 1 >= self.data.len() {
                                break;
                            }
                            let col = &self.data[row - 1];
                            ui.columns(3, |columns| {
                                columns[0].label(&col[0]);
                                columns[1].label(&col[1]);
                                columns[2].label(&col[2]);
                            });
                        }
                    }
                },
            );
```

how to copy the text in the columns?
![image](https://user-images.githubusercontent.com/63939636/189311655-e1047d43-3b39-47da-902a-5d9dabff2cf2.png)

**Comment by ar37-rs** [comment]
Maybe it's good idea to just take a look at `font book` on demo example, if click the emoji symbol it will automatically copied to the clipboard.

**Comment by goldwind-ting** [comment]
it is a little bit difference, I want to drag the cursor to select text in the column.

**Comment by 0xFar5eer** [comment]
I have the same question how to copy text from text_edit ?
I even tried like this, but does not work:
`
if ui.text_edit_singleline(link_to_profile).clicked() {
    ui.output().copied_text = link_to_profile.to_string();
}`

Any ideas how to copy text CTRL + C on text edit?

**Comment by Junoburger** [comment]
Would like this feature as well.

Unless something changed since the last answer in [this thread](https://github.com/emilk/egui/discussions/353)
Selecting (label) text with a mouse and copying it has not been implemented yet.

**Comment by YgorSouza** [comment]
This has been implemented in #3814 and #3870, so I think this issue can be closed.
